e2e/launcher/configuration.go

```diff
@@ -0,0 +1,52 @@
+package launcher
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+)
+
+type NetworkConfiguration struct {
+	NetworkId          string               `json:"network_id"`
+	GenesisPath        string               `json:"genesis_path"`
+	NodeConfigurations []*NodeConfiguration `json:"node_configurations"`
+}
+
+type NodeConfiguration struct {
+	PrivateKey string `json:"private_key"`
+	ConfigPath string `json:"config_path"`
+}
+
+const networkConfigurationFileName = "network.json"
```

Comment:
I see a 😄 emoji but no comments, resolution or change. Will you make the change?

Reply:
The emoji was because this literally made me smile and to remind self that I'd seen it. In this case: network configuration file name is not configurable, so it's not "default", but the "actual" name. It doesn't gel to make it configurable, either, so I won't change it.